packages/legend-studio/src/components/editor/side-bar/WorkspaceRevisionUpdater.tsx

```diff
@@ -0,0 +1,281 @@
+                                key={`conflict-${conflict.entityPath}`}
+                                conflict={conflict}
+                                isSelected={isSelectedConflict(conflict)}
+                                openConflict={openConflict(conflict)}
+                              />
+                            ))}
+                        </div>
+                      </div>
+                    </ResizablePanel>
+                    <ResizablePanelSplitter>
+                      <ResizablePanelSplitterLine color="var(--color-dark-grey-100)" />
+                    </ResizablePanelSplitter>
+                    <ResizablePanel minSize={20}>
+                      <div className="panel workspace-revision-updater__changes__panel">
+                        <div className="panel__header">
+                          <div className="panel__header__title">
+                            <div className="panel__header__title__content">
+                              RESOLVED CHANGES
+                            </div>
+                            <div
+                              className="workspace-revision-updater__changes__panel__title__info"
+                              title="All committed reviews in the project since the revision the workspace is created"
+                            >
+                              <FaInfoCircle />
+                            </div>
+                          </div>
+                          <div className="workspace-revision-updater__changes__panel__header__changes-count">
+                            {changes.length}
+                          </div>
+                        </div>
+                        <div className="panel__content">
+                          {changes
+                            .slice()
+                            .sort(entityDiffSorter)
+                            .map((diff) => (
+                              <EntityDiffSideBarItem
+                                key={diff.key}
+                                diff={diff}
+                                isSelected={isSelectedDiff(diff)}
+                                openDiff={openChange(diff)}
+                              />
+                            ))}
+                        </div>
+                      </div>
+                    </ResizablePanel>
+                  </ResizablePanelGroup>
+                </div>
+              </div>
+            </ResizablePanel>
+            <ResizablePanelSplitter>
+              <ResizablePanelSplitterLine color="var(--color-dark-grey-200)" />
+            </ResizablePanelSplitter>
+            <ResizablePanel>
+              <div className="workspace-revision-updater">
+                <div className="panel__header workspace-revision-updater__header">
+                  <div className="workspace-revision-updater__header__tabs">
+                    {openMergedEditorStates.map((editorState) => (
```

Comment:
I think in the demo video, the tab text color is dark on dark background, let's make sure we fix it